assets/helpers/utilities.js

```diff
@@ -55,6 +57,39 @@ export function parseBoolean(boolString: string, fallback: boolean): boolean {
 
 }
 
-export function titleCase(s: string) {
+function titleCase(s: string) {
   return s.charAt(0).toUpperCase() + s.toLowerCase().substring(1);
 }
+
+// Deserialises a JSON object from a string.
```

Comment:
Is it possible to add test for this?

Reply:
There are several things in the utilities file that don't have tests. I've added a card to bring the whole file up to 100% coverage.